tests/src/Unit/Service/Context/PageContextTest.php

```diff
@@ -1,12 +1,13 @@
 <?php
 
-namespace Drupal\Tests\acquia_lift\Service\Context;
+namespace Drupal\Tests\acquia_lift\Unit\Service\Context;
```

Comment:
All of these "Unit\" additions are not helpful when running PHPUnit straight, but becomes problematic when using Drupal's run-scripts.sh. Had to be fixed.